recipes/kaleido-core/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "kaleido-core" %}
+{% set version = "0.1.0a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_minimal_linux_x64.zip  # [linux64]
+  sha256: 0aa7dde3e767b4b8c1b9aa84de204186442b061fdbeb1ff0c3e486ed7074d7fb  # [linux64]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_mac.zip  # [osx]
+  sha256: 694b7f29a38b7dbc9b262a03a579e56b2ca34ce64ea58ad264b8ef5ae32904cf  # [osx]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_win_x64.zip  # [win64]
+  sha256: b9c80478d684ec25bbece8744c3cd032a7058726c720015479bf5cce7fcfebbf  # [win64]
+
+build:
+  number: 0
+  ignore_run_exports:
+    # Chromium loads sqlite dynamically
+    - sqlite  # [linux64]
+  missing_dso_whitelist:
+    - /usr/lib/*  # [osx]
+    - /System/Library/Frameworks/*  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # Uncomment after feedstock creation to specify minimum GLIBC version and change
+    # docker image to condaforge/linux-anvil-cos7-x86_64
+    # - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - expat   # [linux64]
+    - nspr    # [linux64]
+    - nss     # [linux64]
+    - sqlite  # [linux64]
+  run:
+    - expat   # [linux64]
+    - nspr    # [linux64]
+    - nss     # [linux64]
+    - sqlite  # [linux64]
+    - fontconfig         # [linux64]
+    - fonts-conda-forge  # [linux64]
```

Comment:
Should these just be `linux`?

Reply:
Thanks, done in https://github.com/conda-forge/staged-recipes/pull/12747/commits/e4c9ee22aa36a5b05aa0c0541879bb3920b92962